### DISCUSSION TITLE
删除数据库之后多久内存会恢复。

### DISCUSSION THREAD

**Discussion by susoooon** [milvus-io/milvus#16872] [Q&A and General discussion]
2.0.2，pymilvus在代码里清空了所有的数据，并且删除了collection。但是在服务器上看内存仍然有1..5G左右占用。十分钟之后也没有变化，重启docker之后内存恢复正常。我应该设置哪个配置文件的参数可以更快的释放掉这部分内存

**Comment by xiaofan-luan** [comment]
The go memory can not be released.
If you have a memory analysis graph would be helpful

**Reply by susoooon** [reply]
“The go memory can not be released.”，这意味着我的内存会越来越大直到爆炸？事实上我发现更多的向量在删除后，会导致更多的内存占用，在多次的删除和新建操作之后内存越来越大

**Comment by chuzhifeng** [comment]
我也遇到了这个问题，除了重启docker，还有其他更好的方式处理吗

**Comment by yhmo** [comment]
2.0.2有一些内存泄漏和内存使用不合理的问题，我门在2.1中做了修复，请等待2.1的发布。

PS：我们鼓励用英文提问哈
